x/stake/keeper/key_test.go

```diff
@@ -0,0 +1,35 @@
+package keeper
+
+import (
+	"encoding/hex"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/stake/types"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+)
+
+var (
+	pk1 = ed25519.GenPrivKey().PubKey()
+)
+
```

Comment:
Do we have unit tests for `GetREDByValSrcIndexKey ` and `GetREDByValDstIndexKey`? Otherwise, this LGTM. Thanks 👍

Reply:
We don't have any to my knowledge. I can make some

Reply:
Let's do that, but it can be in a separate PR.